src/bin/create_user.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('commander');
+const Web3 = require('web3');
+const OpenST = require('@openstfoundation/openst.js');
+
+const connected = require('../connected');
+const logger = require('../logger');
+const { version } = require('../../package.json');
+
+const { GnosisSafe } = OpenST.ContractInteract;
+
+program
+  .version(version)
+  .name('Create Wallet User')
+  .arguments('<config> <eip20Token> <owners> <threshold> <sessionKeys> <sessionKeySpendingLimits> <sessionKeyExpirationHeights>')
+  .description('An executable to register rules in tokenrules.')
+  .action(
+    async (
+      config,
+      eip20Token,
+      owners,
+      threshold,
+      sessionKeys,
+      sessionKeySpendingLimits,
+      sessionKeyExpirationHeights,
+    ) => {
+      await connected.run(
+        config,
+        async (chainConfig, connection) => {
+          const createUserTxOptions = {
+            from: connection.auxiliaryAccount.address,
+            gasPrice: chainConfig.auxiliaryGasPrice,
+          };
+          const auxiliaryWeb3 = new Web3(connection.auxiliaryWeb3);
+          const userHelper = new OpenST.Helpers.User(
+            chainConfig.openst.tokenHolderMasterCopy,
+            chainConfig.openst.gnosisSafeMasterCopy,
+            chainConfig.openst.recoveryMasterCopy,
+            chainConfig.openst.createAndAddModules,
+            eip20Token,
+            chainConfig.openst.tokenRules,
+            chainConfig.openst.userWalletFactory,
+            chainConfig.openst.proxyFactory,
+            auxiliaryWeb3,
+          );
+          const ownersArray = owners.split(',').map(item => item.trim());
+          const sessionKeysArray = sessionKeys.split(',').map(item => item.trim());
+          const sessionKeySpendingLimitsArray = sessionKeySpendingLimits.split(',').map(item => item.trim());
+          const sessionKeyExpirationHeightsArray = sessionKeyExpirationHeights.split(',').map(item => item.trim());
+          const response = await userHelper.createUserWallet(
+            ownersArray,
+            threshold,
+            chainConfig.openst.recoveryOwnerAddress,
+            chainConfig.openst.recoveryControllerAddress,
+            chainConfig.openst.recoveryBlockDelay,
+            sessionKeysArray,
+            sessionKeySpendingLimitsArray,
+            sessionKeyExpirationHeightsArray,
+            createUserTxOptions,
+          );
+          const { returnValues } = response.events.UserWalletCreated;
+          const gnosisSafeProxy = returnValues._gnosisSafeProxy;
```

Comment:
Do you not want to write these value back in the config?

Reply:
Good Point 💯 Will update!